File: templates/java/fr/text/code/cornerBeepers10.java
/*
 * Fichier: CoinCinqJetons.java
 * ----------------------------
 * Place cinq jetons dans chaque coin
 */
import stanford.karel.*;
public class CoinCinqJetons extends Karel {

   public void run() {
      /* Répétez une fois pour chaque coin */
      for(int i = 0; i < 4; i++) {
         mettreCinqJetons();
         avancerAuCoinSuivant();
      }
   } 

   /* repositionner Karel au coin suivant */
   public void avancerAuCoinSuivant() {
      avancer();
      avancer();
      avancer();
      tournerGauche();
   }

   /* place 5 jetons utilisant un boucle for */
   public void mettreCinqJetons() {
      for(int i = 0; i < 5; i++) {
      	laisserJeton();
      }
   }
}

File: templates/python/fr/text/code/cornerBeepers10.py
# Fichier: CornerFive Jetons .py
# -----------------------------
# Place cinq jetons dans chaque coin
from karel.stanfordkarel import *
def main():
   # Répétez une fois pour chaque coin
   for i in range(4):
      mettre_cinq_jetons()
      avancer_au_coin_suivant()

# repositionner Karel dans le coin suivant
def avancer_au_coin_suivant() :
   avancer()
   avancer()
   avancer()
   tourner_gauche()

# places 5 jetons utilisant un boucle for
def mettre_cinq_jetons() :
   for i in range(5):
   	laisser_jeton()
